app/actions/WalletLoaderActions.js

```diff
@@ -415,7 +415,10 @@ export const spvSyncAttempt = (privPass) => (dispatch, getState) => {
   const { discoverAccountsComplete, spvConnect } = getState().walletLoader;
   var request = new SpvSyncRequest();
   if (spvConnect) {
-    request.setSpvConnect(spvConnect);
+    var splitSpvConnect = spvConnect.split(",");
```

Comment:
hmmm, could the incoming `spvConnect` here be an array? I believe it's coming from an exported config for the moment, so if feasible, let's use an array as it avoids this kinds of hacky things

Reply:
that's true.  I'll make that change with the config